src/pages/debug-app/debug-app.tsx

```diff
@@ -861,7 +829,6 @@ export const DebugApp = () => {
           <DebugPanel
             onClose={() => onChangeMainToolsPanelHandler(ActiveButton.debug)}
             debugOptions={debugOptions}
```

Comment:
This prop is not needed

Reply:
1. removed debugOptions prop, from DebugPanel in the debug-app.tsx and debug-panel.tsx
2. added const debugOptions = useAppSelector(selectDebugOptions); in the debug-panel.tsx